packages/server/test/unit/modes/run_spec.coffee

```diff
@@ -288,6 +288,7 @@ describe "lib/modes/run", ->
   context ".waitForTestsToFinishRunning", ->
     beforeEach ->
       sinon.stub(@projectInstance, "getConfig").resolves({})
+      sinon.stub(runMode, "_delayToLetVideoFinish").resolves()
```

Comment:
why not stub the delay value instead of the function?

Reply:
well, i guess you would need to `spy` on the function either way

Reply:
yeah, making it a method allows spying and stubbing on it during tests, rather than exposing a constant - plus the logic for waiting could be dynamic in the future (like waiting for a browser event for example)